### DISCUSSION TITLE
I can't read images in an excel file

### DISCUSSION THREAD

**Discussion by shaiasf** [PHPOffice/PhpSpreadsheet#2887] [Q&A] [answered]
I'm working with an excel file (.xlsx) that contains images and using getDrawingCollection() to read the images, but it returns an empty array. Don't know what I might be doing wrong?

**Comment by oleibman** [comment]
Might you have multiple sheets and be requesting the drawing collection from the "wrong" sheet?

**Comment by Zjmainstay** [comment]
@oleibman 
The insert image can not read by getDrawingCollection, it got data like =DISPIMG("ID_19EA7EA06F264FC8BCE74B988F9DF2B0",1)
see the attachment.

[read_excel.zip](https://github.com/user-attachments/files/16099388/read_excel.zip)

**Comment by MarkBaker** [comment, marked as answer]
I thought DISPIMG() was a WPS function, AFAIK it's not an Excel function. MS online has no documentation on any function with that name; and the new Excel IMAGE() function is not yet implemented in PhpSpreadsheet

**Reply by Zjmainstay** [reply]
ok, thanks.
